### PR TITLE
[4.1] Check if the output to escape is null in BaseLayout

### PR DESCRIPTION
**Pull request by laoneo**
### Summary of Changes
When a contact is edited, the form shows a deprecated notice around the user field in PHP 8.1

### Testing Instructions
Create a contact without a linked user and edit it.

### Actual result BEFORE applying this Pull Request
The following error is shown:
_Deprecated: htmlspecialchars(): Passing null to parameter #1 ($string) of type string is deprecated in /libraries/src/Layout/BaseLayout.php on line 119_

### Expected result AFTER applying this Pull Request
No error.

### CODE REVIEW

**Comment by dgrammatiko** [comment]
I have tested this item :white_check_mark: successfully on 1ac32ade36b264c2db7de735947154f8d96ad332<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37333">issues.joomla.org/tracker/joomla-cms/37333</a>.</sub>

File: libraries/src/Layout/BaseLayout.php [hunk 1]
```diff
@@ -116,6 +116,11 @@ public function resetOptions()
 	 */
 	public function escape($output)
 	{
+		if ($output === null)
```
Comment (dgrammatiko) [inline libraries/src/Layout/BaseLayout.php]: fwiw I would prefer a single line for all these, something like:
```php
return $output === null ? '' : htmlspecialchars($output, ENT_QUOTES, 'UTF-8');
```

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 1ac32ade36b264c2db7de735947154f8d96ad332<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37333">issues.joomla.org/tracker/joomla-cms/37333</a>.</sub>

**Comment by laoneo** [comment]
rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37333">issues.joomla.org/tracker/joomla-cms/37333</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**